README.md

```diff
@@ -440,6 +440,31 @@ Do not leave console calls in your code
 
 > Reason: it just a noise for other developers. We always can find anything using history of version control system.
 
+### Specify (`TODO` | `FIXME`) comment with JIRA (or another bug tracking system) task id or username
+
+> Reason: to not keep not evident and forgotten comments please specify a comment with a task id which is connected with a problem. If it's a light problem and you are going to solve it soon you can provide just your username. 
```

Comment:
I guess we could add a few more words here. It would be better to describe a little bit what is TODO and how useful could it be. I guess it will help the readers.

TODO is a useful tool for developers.
There are a few cases when TODO comments could be added:
* the developer is working on the task, and he noticed that part of his work is temporarily blocked (say, his particular task depends on API that not yet ready)
* the developer is working on the task and noticed that he would like to make one/few little improvements later in the scope of this task. However, the task is already done and could be sent for code-review to the boost a process.
* the developer detects some issue in the code and unable to fix it in the scope of his task, but he would like to improve the situation.

**Do not leave anonymous TODO without the id**. Usually, anonymous TODO's live too long while it not linked to the issue tracking system. 
Signed TODO's increases the factor of responsibility. When the developer signs TODO it like a short-term note for the author. But, it also a flag that author is responsible to resolve it ASAP. Finally, it makes such part of code clear for reviewers.

Reply:
It's good notes, but I think it's too huge to add it to our style guide and most developers should know about it. Maybe it's better to add more examples instead of

Reply:
add this text under a spoiler

Reply:
great, thank you!